### PR TITLE
[v6r19] Replace gLogger and fix some issues due to the changes

### PR DESCRIPTION
**Pull request by aldbr**
This pull request just replaces the logging system *gLogger* by the new one and fix some issues due to the changes.

### CODE REVIEW

File: Interfaces/API/Job.py [hunk 1]
```diff
@@ -1223,4 +1223,27 @@ def runLocal( self, dirac = None ):
 
     return dirac.submitJob( self, mode = 'local' )
 
+  #############################################################################
+
+  def __getstate__(self):
```
Comment (chaen) [inline Interfaces/API/Job.py]: This is not from you. Why is it in your PR ?

Comment (aldbr) [inline Interfaces/API/Job.py]: I updated the existing methods in API

**Comment by chaen** [comment]
OK for me. Maybe @andresailer wants to check this getstate method ?

File: Core/Base/API.py [hunk 2]
```diff
@@ -65,21 +65,31 @@ def __init__( self ):
...
-    self.__dict__.update(d)
-    if isinstance( d['log'], basestring ):
-      self.log = gLogger.getSubLogger( d['log'] )
+    :return: dictionary of attributes
+    """
+    from DIRAC.FrameworkSystem.private.standardLogging.Logging import Logging
+    state = dict(self.__dict__)
+    # Replace the Logging instance by its name because it is not copyable
+    # because of the thread locks
+    if isinstance(state['log'], Logging):
```
Comment (andresailer) [inline Core/Base/API.py]: To keep this backward compatible, also check for SubSystemLogger instance (keeping the other import)
```
if isinstance(state['log'], (SubSystemLogger, Logging) ):
```

**Comment by andresailer** [comment]
`__setstate__` seems to only differ in whitespace and variable names

File: Core/Base/API.py (hunk 2 above)
Comment (aldbr) [inline Core/Base/API.py]: Ok I kept these lines

**Pull request merged**